.travis.yml

```diff
@@ -10,14 +9,29 @@ cache:
   directories:
   - $HOME/.gradle/caches/
   - $HOME/.gradle/wrapper/
-
-jdk:
-  - oraclejdk8
-  - openjdk8
-  - oraclejdk9
+  - $HOME/.android/build-cache
 
 before_install:
   - pip install --user codecov
 
-after_success:
-- codecov
+matrix:
+  include:
+    - language: java
+      jdk: oraclejdk8
+      after_success:
+        - codecov
```

Comment:
I think running codecov once should suffice actually :)

Reply:
Fixed